setup.py

```diff
@@ -55,6 +55,7 @@ def run(self):
 
 install_requires = [
     "alembic>=0.6.6",
+    "anyjson>=0.3.3",
```

Comment:
@Dziolas because nooneelse reported similar problem, please remove `anyjson` from the `setup.py`.